test/lib/jdk/test/lib/Platform.java

```diff
@@ -60,12 +60,17 @@ private static String privilegedGetProperty(String key) {
                 PrivilegedAction<String>) () -> System.getProperty(key));
     }
 
+    private static boolean isJ9() {
+        return vmName.toLowerCase().contains("openj9")
+               || vmName.toLowerCase().contains("ibm");
```

Comment:
The uses of `toLowerCase()` are not necessary; just look for 'OpenJ9' or 'IBM'.

Reply:
Updated.